src/lib/common/Button.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+    export let color: string, hover: string, href: string, text: string, width: string;
```

Comment:
Imo, I don't think we should pass full-fledged tailwind classes as props. For example, typing `width="w-32"` feels really cumbersome, why do have to write width twice?

I'm giving the following suggestions, see if you like them:
- For color it works well because I just type color={color}. Since we the "hover" for a secondary color, I would change the prop to `secondaryColor` or `hoverColor` (or wtv you want) and simply pass the color too. When using the prop, it'd become something like  `hover:{secondaryColor}`
- For width, we could perhaps take on a similar strategy. I also thought about having some semantic prop like "large" or "small" but then I didn't like adding abstraction on top of tailwind
- href shouldn't be mandatory, we don't always want to redirect the user when clicking a button. I'd even consider removing `<a>` from the component but that's your call

Reply:
I thought about doing this, but they recommend against it in the docs: (https://tailwindcss.com/docs/content-configuration#dynamic-class-names). In fact, I tried to do it now in the width (passing 96 and then using w-{width}) and it didn't work. The solution would be to map keywords to their corresponding class in Tailwind, but since you don't like the abstraction of using "large" and "small", I'm guessing that's not an option.

Reply:
I noticed, however, that I'm doing the exact same thing with the "align" property on the ErrorSection component and it works fine. Strange...

Reply:
Nevermind, I found out why: the first section already uses `text-center` and the footer uses `text-right`, that's why it works when I change any of them.

Reply:
I see, I wasn't aware of that (since I'm not experienced with tailwind). I'd suggest following the docs then

Reply:
Okay. Just to clarify: would you rather have the classes mapped to keywords or write the classes themselves when calling the components? It's a tradeoff between ease of use and versatility, I guess.

Reply:
You're right, it's a tradeoff. Maybe I'd go for the keywords but it's your call